apps/meteor/app/livechat/server/livechat.ts

```diff
@@ -18,24 +23,39 @@ WebApp.connectHandlers.use('/livechat', (req, res, next) => {
 	}
 
 	res.setHeader('content-type', 'text/html; charset=utf-8');
-
+	const liveChatAdditionalScripts = settings.get<string>('Livechat_AdditionalWidgetScripts');
```

Comment:
When I try to use an additional script I get this error

![Screenshot 2023-12-13 at 14 26 28](https://github.com/RocketChat/Rocket.Chat/assets/20868078/b280a5fa-6dca-4823-9dfb-24923ac0de4a)

Do we need a specific setting to use this (to embed livechat we usually have to change some CORS settings)

Reply:
How were you trying to use the script ?  i want to verify that on my end.

Reply:
I was trying to use javascript directly, now I see that there we should be adding just the src there, we should probably specify that the src property of the tag will be the one set by the setting

Reply:
I verified using the javascript elements in the iframe inside the livechat widget.

<img width="1483" alt="image" src="https://github.com/RocketChat/Rocket.Chat/assets/213379/c89386ab-9ac6-4386-b480-a3692722f7d4">

Reply:
https://github.com/RocketChat/Rocket.Chat/assets/213379/c9a6bd61-42ff-46fa-ad51-6b6b2a4581f6

Showing the scripts and css in use with an example of datatables

Reply:
Corrected the issue with multiple-times loading the script. Please check it again.